to.etc.domui/src/main/java/to/etc/domui/server/AppFilter.java

```diff
@@ -90,6 +93,11 @@ static public synchronized void setIoWrapper(@NonNull IRequestResponseWrapper ww
 		m_ioWrapper = ww;
 	}
 
+	@Nullable
+	public static Boolean isProductionMode() {
```

Comment:
Should be isTestMode, and most certainly not be nullable- that does not mean anything.